source/guide_tiddlywiki5.rst

```diff
@@ -0,0 +1,187 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. tag:: wiki
+.. tag:: web
+.. tag:: lang-javascript
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/tiddlywiki.svg
+      :align: center
+
+############
+TiddlyWiki 5
+############
+
+.. tag_list::
+
+TiddlyWiki_ is a personal wiki and a non-linear notebook for organising and sharing complex information.
+
+License
+=======
+
+TiddlyWiki_ is published under a `permissive BSD 3-Clause License`_.
+
+Prerequisites
+=============
+
+If you want to use TiddlyWiki with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Step 1
+------
+TiddlyWiki is installed via Nodes package manager `npm`:
+
+::
+
+ [isabell@stardust ~]$ npm install -g tiddlywiki
+ [...]
+ + tiddlywiki@5.1.23
+ added 1 package from 1 contributor in 5.102s
+ [isabell@stardust ~]$ 
+
+Check if TiddlyWiki is installed by typing:
+
+::
+
+ [isabell@stardust ~]$ tiddlywiki --version
+ 5.1.23
+ [isabell@stardust ~]$
+
+Step 2
+------
+The following commands creates a folder in your home directory for TiddlyWiki. After that, a new wiki folder (named ``mynewwiki`` in this example) containing server-related components is created.
+
+::
+
+ [isabell@stardust ~]$ mkdir tiddlywiki
+ [isabell@stardust ~]$ cd tiddlywiki
+ [isabell@stardust tiddlywiki]$ tiddlywiki mynewwiki --init server
+ Copied edition 'server' to mynewwiki
+ [isabell@stardust tiddlywiki]$ ls
+ mynewwiki
+ [isabell@stardust tiddlywiki]$
+
+Step 3
+------
+
+.. note::
+
+    TiddlyWiki is running on port 8080.
+
+.. include:: includes/web-backend.rst
+
+Step 4
+------
+
+Now TiddlyWiki server can be started:
```

Comment:
What do you think about skipping this step and moving "Step 5" after "Step 6"? Or is there some benefit to starting the server manually first that I am not getting?